playwright.config.ts

```diff
@@ -43,52 +43,47 @@ export default defineConfig({
   /* Configure projects for major browsers */
   projects: [
     {
-      name: 'chromium',
+      name: 'chromium-login',
+      testMatch: /login\.spec\.ts$/,
       use: {
         ...devices['Desktop Chrome'],
       },
     },
     {
-      name: 'firefox',
+      name: 'chromium-email-submission',
+      testMatch: /email-submission\.spec\.ts$/,
```

Comment:
@opengovsg/formsg-engineers note that for playwright, we're adding hardcoded tests suites. i.e., new test suites permutation will need to be added here, which shouldn't happen as we want to deprecate playwright in favour of chromatic / FE unit tests.